Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php

```diff
@@ -34,13 +34,13 @@
  *
  * @api Note: The constructor is not part of the public API
  */
-final class Node
+final readonly class Node
```

Comment:
I remember very loosely, that @bwaidelich had some issues with readonly classes and proxy builder. 🤷‍♂️ But I could be totally wrong.

Reply:
@robertlemke luckily fixed these issues now ^^ (see testcases in flow :D)